resources/legacy_redirects.conf

```diff
@@ -1079,4 +1079,5 @@ rewrite (?i)^/guide/en/beats/filebeat/7.0/filebeat-module-apache2.html /guide/en
 rewrite (?i)^/guide/en/elasticsearch/reference/master/data-frame-transform-pivot.html /guide/en/elasticsearch/reference/master/data-frame-transform.html permanent;
 rewrite (?i)^/guide/en/elasticsearch/reference/7.x/data-frame-transform-pivot.html /guide/en/elasticsearch/reference/7.x/data-frame-transform.html permanent;
 rewrite (?i)^/guide/en/elasticsearch/reference/7.3/data-frame-transform-pivot.html /guide/en/elasticsearch/reference/7.3/data-frame-transform.html permanent;
-rewrite (?i)^/guide/en/elasticsearch/reference/7.2/data-frame-transform-pivot.html /guide/en/elasticsearch/reference/7.2/data-frame-transform.html permanent;
\ No newline at end of file
+rewrite (?i)^/guide/en/elasticsearch/reference/7.2/data-frame-transform-pivot.html /guide/en/elasticsearch/reference/7.2/data-frame-transform.html permanent;
+rewrite (?i)^/guide/en/apm/agent/js-base/(.*) /guide/en/apm/agent/rum-js/$1;
```

Comment:
@nik9000 👍/ 👎?

Reply:
Could you add a new line on the end of this? I just like having trailing newlines. Looks like I didn't have one before. Ooops!

Otherwise :+1:

Reply:
Done. Thank you!